scripts/utils.js

```diff
@@ -0,0 +1,123 @@
+const path = require('path');
+const fs = require('fs-extra');
+const colors = require('colors/safe');
+const green = colors.green;
+const blue = colors.blue;
+const cyan = colors.cyan;
+const yellow = colors.yellow;
+const isWindows = /^win/.test(process.platform);
+
+const ROOT = path.resolve(__dirname, '..');
+const NODE_ENV = process.env.NODE_ENV;
+
+
+function onError(err) {
+	console.log(colors.red('Error:'), err);
+}
+
+function onSuccess(result) {
+	var msg = `${green('Success:')} ${blue(`[${result.action}]`)} ${result.count} files processed`;
+	if(result.totalCount) {
+		msg += ` (out of total ${result.totalCount} matched)`; 
+	}
+
+	msg += ` [${yellow(`${result.processingTime}ms`)}]`;	
+
+	console.log(msg);
+}
+
+function onProgress(sourcefile, outfile, operation) {
+	if('isError' in global && global.isError) {
+		return;
+	}
+	console.log(`${colors.blue(`[${operation}]`)} ${sourcefile} -> ${outfile}`);
```

Comment:
I think we can remove the ` -> ${outfile}`. All it's ever adding is `build/` at the beginning, right? Easier to read without that.